python-doctest/fizz_buzz.py

```diff
@@ -0,0 +1,73 @@
+# Replace numbers that are divisible by 3 with "fizz"
+def fizz_buzz(numbers):
+    """Implement the Fizz buzz game.
+
+    >>> fizz_buzz([3, 6, 9, 12])
+    ['fizz', 'fizz', 'fizz', 'fizz']
+    """
+
+
+# # Replace numbers that are divisible by 3 with "fizz"
+# def fizz_buzz(numbers):
+#     """Implement the Fizz buzz game.
+
+#     >>> fizz_buzz([3, 6, 9, 12])
+#     ['fizz', 'fizz', 'fizz', 'fizz']
+#     """
+#     result = []
+#     for number in numbers:
+#         if number % 3 == 0:
+#             result.append("fizz")
+#         else:
+#             result.append(number)
+#     return result
+
+
+# Replace numbers that are divisible by 5 with "buzz"
+# def fizz_buzz(numbers):
+#     """Implement the Fizz buzz game.
+
+#     >>> fizz_buzz([3, 6, 9, 12])
+#     ['fizz', 'fizz', 'fizz', 'fizz']
+
+#     >>> fizz_buzz([5, 10, 20, 25])
+#     ['buzz', 'buzz', 'buzz', 'buzz']
+#     """
+#     result = []
+#     for number in numbers:
+#         if number % 3 == 0:
+#             result.append("fizz")
+#         elif number % 5 == 0:
+#             result.append("buzz")
+#         else:
+#             result.append(number)
+#     return result
+
+
+# Replace numbers that are divisible by 3 and 5 with "fizz buzz"
```

Comment:
I think in this case it makes sense to leave the final implementation uncommented, and leave a comment saying that the commented versions are previous stages, that said, I think you can leave out the previous stages too.